lib/core/src/Cardano/Wallet/Transaction.hs

```diff
@@ -100,6 +100,26 @@ data TransactionLayer k = TransactionLayer
         -- This expects as a first argument a mean to compute or lookup private
         -- key corresponding to a particular address.
 
+    , mkUnsignedTransaction
+        :: AnyCardanoEra
+            -- Era for which the transaction should be created.
+        -> XPrv
```

Comment:
I think only an XPub is needed here, right?

Reply:
indeed, used XPub rather than XPrv